mobile/test/performance/files_em_does_not_use

```diff
@@ -5,7 +5,6 @@ source/extensions/filters/http/on_demand/config.h
 source/extensions/filters/http/on_demand/on_demand_update.h
 source/common/http/route_config_update_requster.h
 source/common/listener_manager/listener_manager_impl.h
-source/common/network/io_uring_socket_handle_impl.h
 source/server/overload_manager_impl.cc
```

Comment:
io_uring_socket_handle_impl.h will be removed on mobile platform according to the `//bazel:android`, while mobile/perf runs on Linux. Add the file back will not increase binary size on iOS and Android.
https://github.com/envoyproxy/envoy/blob/2086a804671ff922666b99467c6ccf141ff8729e/source/common/network/BUILD#L262-L268